source/includes/fact-windows-unattended.rst

```diff
@@ -0,0 +1,33 @@
+Install Windows Unattended
```

Comment:
Should be clear to Windows users; commonplace terminology in the Windows ecosystem.